src/apps/material/material.dev.tsx

```diff
@@ -26,6 +26,16 @@ export default {
       defaultValue: "1",
       control: { type: "text" }
     },
+    blacklistedBranchesConfig: {
+      name: "Blacklisted branches",
+      defaultValue: "FBS-751032,FBS-751031,FBS-751009,FBS-751027,FBS-751024",
+      control: { type: "text" }
+    },
```

Comment:
Consider if it is worth specifying that this blacklist is for reservation pickup locations.

There will be other blacklists in future PRs. If future PRs rename the entry you can disregard this.